### ISSUE TITLE
Weird error in CoCoSpec mode require variable compilation

### ISSUE THREAD

**Issue by hbourbouh** [kind2-mc/kind2#624]
Hi Daniel,

Attached is an exemple of a Lustre model that returns the following error:
```
kind2 --modular true test8_ERROR_2_PP.KIND2.lus
 kind2 nightly

<Error> Error opening input file 'test8_ERROR_2_PP.KIND2.lus': Not_found
```

Once I change the order of equation on line 45 to line 48. The error disappears.
Also if I remove the variable  "art_1" and use the node call directly in the mode, the error disappears.

Do you have an idea why the order of Lustre equations is important?

Thanks,
Hamza
[test8_ERROR_2_PP.KIND2.lus.txt](https://github.com/kind2-mc/kind2/files/4404280/test8_ERROR_2_PP.KIND2.lus.txt)

**Comment by wangte1987** [comment]
Hi Daniel, 

Any chance of diagnosing this issue? I just compiled again from the latest developer's branch and I still have this problem.  An older compile kind2-v1.1.0-292-gad768f58 from does not? 

Thanks, 

Tim

**Comment by daniel-larraz** [comment]
Commit f2ea7a1, included in version 1.5.0, fixed this issue.
